app/assets/javascripts/modules/CaseClosure.js

```diff
@@ -52,6 +61,17 @@ moj.Modules.CaseClosure = {
     });
   },
 
+  showHideOtherOverturned: function() {
+      var other_is_checked = this.$outcomeReasonOtherOption.is(':checked');
+      var other_overturned_has_value = $('#sar_internal_review_other_overturned').val() ? true : false;
+
+      if (other_is_checked || other_overturned_has_value) {
```

Comment:
Just a question here, should it show the "Other" input if it's not checked, just because there's a value in it?

Reply:
Yes that is a good point - it could be that they uncheck "other" when the input still has a value in it.